internal/cmd/commands/server/server.go

```diff
@@ -667,69 +667,84 @@ func (c *Command) StartWorker() error {
 
 func (c *Command) WaitForInterrupt() int {
 	const op = "server.(Command).WaitForInterrupt"
-	// Wait for shutdown
-	shutdownTriggered := false
 
-	// Add a force-shutdown goroutine to consume another interrupt
-	abortForceShutdownCh := make(chan struct{})
-	defer close(abortForceShutdownCh)
+	var shutdownCompleted atm.Bool
+	shutdownTriggerCount := uint32(0)
```

Comment:
With all of the more recent changes I do not think it's necessary for shutdownTriggerCount to be an atomic any more, because the goroutines are kicked off _after_ reading the value from the main loop.

If you want to keep it atomic, note that you shouldn't just declare a uint32 and run atomic functions on it anymore in Go 1.19; use `atomic.NewUint32` instead.

Reply:
Done!